h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -83,15 +83,17 @@ protected ModelBuilder(P parms, Job<M> job) {
    *  default settings. */
   protected ModelBuilder(P parms, boolean startup_once) { this(parms,startup_once,"hex.schemas."); }
   protected ModelBuilder(P parms, boolean startup_once, String externalSchemaDirectory ) {
-    assert startup_once;
+    String base = getClass().getSimpleName().toLowerCase();
+    if (! startup_once)
```

Comment:
tiny style thing - `!startup_once` ?

Reply:
yeah :) IntelliJ makes the same point as you do, will fix